src/moai-sim/MOAIDrawShape.cpp

```diff
@@ -170,7 +170,10 @@ void MOAIDrawShape::DrawPoint ( const ZLVec2D& loc ) {
 //----------------------------------------------------------------//
 void MOAIDrawShape::DrawPoint ( float x, float y, float z ) {
 
-	this->DrawPoint ( x, y, z );
```

Comment:
@patrickmeehan this is the same weird logic where it tries to call itself. I just do nothing here to make the compiler happy, but maybe you know what to do to actually get it to "draw a point"

Reply:
@david-pershouse My bad, same as the other. Should be MOAIDrawShape_DrawPoint ().